### PR TITLE
GRAPH-902 Helm Chart for Aerospike Graph 

### PR DESCRIPTION
**Pull request by cinterloper**
This PR provides a basic Helm Chart for Aerospike Graph

README by Steve Renaker

### CODE REVIEW

File: aerospike-graph/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- A running instance of [Aerospike Database Server](https://aerospike.com/docs/database)
+  with an accessible IP address and port.
+- [Kubernetes](https://kubernetes.io/) and [Helm](https://helm.sh/).
+- The scripts in the `helm/graphservice` directory.
+## Usage
+The arguments you pass to the `helm` command depend on your Aerospike Server
+network and namespace configuration. The prototype form of the command is as
+follows:
+```bash noCopy
+$ helm [INSTALL|UPGRADE] [POD-NAME] helm/graphservice \
```
Comment (abhishekdwivedi3060) [inline aerospike-graph/README.md:17]: nit:
```suggestion
$ helm [INSTALL|UPGRADE] [RELEASE-NAME] helm/graphservice \
```
In helm's terminology it's called release name.

File: aerospike-graph/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+If you're just getting started with trying out [Aerospike Graph Service](https://aerospike.com/docs/graph)
+(AGS), you may wish to start with a local deployment for testing purposes. When you're ready to
+deploy AGS in a production environment, you'll probably want to use
+containerization and orchestration tools. This guide is intended to illustrate deploying AGS
+with [Kubernetes](https://kubernetes.io/) and [Helm](https://helm.sh/).
+## Prerequisites
+- A running instance of [Aerospike Database Server](https://aerospike.com/docs/database)
+  with an accessible IP address and port.
+- [Kubernetes](https://kubernetes.io/) and [Helm](https://helm.sh/).
+- The scripts in the `helm/graphservice` directory.
```
Comment (abhishekdwivedi3060) [inline aerospike-graph/README.md:11]: You would need steps here on how to add a helm repo containing graphservice helm-chart.
Refer existing charts README

File: aerospike-graph/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+```
+Once your AGS instance is up and running, you can connect to it over
+the [Gremlin websocket port](https://tinkerpop.apache.org/docs/3.6.4/reference/#connecting-gremlin-server).
+For more information about using the Gremlin console with AGS, see
+[Basic Graph Usage](https://aerospike.com/docs/graph/getting-started/basic-usage).
+You can get networking information about your active pods with the `get services` command:
+```ascii
+$ kubectl get services
+NAME                               TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+aerospike-graph-pod-graphservice   LoadBalancer   10.107.38.223   10.107.38.223   8182:32330/TCP   5m22s
```
Comment (abhishekdwivedi3060) [inline aerospike-graph/README.md:65]: The example given and its output (resource naming) are inconsistent. See if we can keep them consistent

File: aerospike-graph/values.yaml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: LoadBalancer
+  port: 8182
+
+ingress:
+  enabled: true
+  className: ""
```
Comment (abhishekdwivedi3060) [inline aerospike-graph/values.yaml:50]: Service type LB and Ingress are 2 different ways of exposing a Kubernetes application to the outside world.
Do we need to enable both of simultaneously? If not we can set `ingress.enabled: false` by default and use LB service instead

**Review by abhishekdwivedi3060 — CHANGES_REQUESTED** [review]
If you want to publish this chart in the aerospike helm repo, you would need to do the following as well:
1. tar this helm-chart and copy it to `docs` dir . (helm package command)
2. Update the `index.yaml` in `docs` dir. (helm repo index command)

**Comment by arrowplum** [comment]
@cinterloper are you wanting to publish to our helm repo or just have the chart available? If the former see the above comment from @abhishekdwivedi3060

File: aerospike-graph/README.md (hunk 3 above)
Comment (cinterloper) [reply aerospike-graph/README.md]: Readme updated from `aerospike-graph-pod-graphservice` to `test-pod-graphservice`

File: aerospike-graph/values.yaml (hunk 4 above)
Comment (cinterloper) [reply aerospike-graph/values.yaml:50]: Removed ingress configuration and set to false. Tested with `minikube tunnel`.

**Comment by cinterloper** [comment]
> @cinterloper are you wanting to publish to our helm repo or just have the chart available? If the former see the above comment from @abhishekdwivedi3060

Do want to publish to the repo

**Comment by cinterloper** [comment]
> If you want to publish this chart in the aerospike helm repo, you would need to do the following as well:
> 
> 1. tar this helm-chart and copy it to `docs` dir . (helm package command)
> 2. Update the `index.yaml` in `docs` dir. (helm repo index command)

Tared and repo index updated.

File: aerospike-graph/README.md (hunk 2 above)
Comment (cinterloper) [reply aerospike-graph/README.md]: @abhishekdwivedi3060 thanks for adding this here 
https://github.com/aerospike/helm-charts/pull/10/files#diff-dc975df3ba4e6d881140069077dd51c454470a8dd9327c8fad3d1240d62ed9d8R15

**Pull request merged**